src/gui/folderman.cpp

```diff
@@ -292,6 +292,21 @@ void FolderMan::setupFoldersHelper(QSettings &settings, AccountStatePtr account,
                 SyncJournalDb::maybeMigrateDb(folderDefinition.localPath, folderDefinition.absoluteJournalPath());
             }
 
+#if defined ENFORCE_VIRTUAL_FILES_SYNC_FOLDER && ENFORCE_VIRTUAL_FILES_SYNC_FOLDER
```

Comment:
Why the && ?

Reply:
Would it be possible to make that a regular if statement? The compiler should be able to optimize it. Also, this block could be extracted in it's own method. Then `switchToVfs` can be const in any case

Reply:
OK I forgot that `cmakedefine01` will always define the variable either to `0` or to `1`
indeed I could use it in a regular `if`